docker-compose.yml

```diff
@@ -30,5 +34,19 @@ services:
       - POSTGRES_PASSWORD=postgres
       - POSTGRES_DB=url-shortener
 
+  redis:
+    image: "redis:alpine"
+    container_name: redis
+    command: redis-server --requirepass redis
+    ports:
+      - "6379:6379"
+    volumes:
+      - redis-data:/var/lib/redis
+      - redis-conf:/usr/local/etc/redis/redis.conf
+    environment:
+      - REDIS_REPLICATION_MODE=master
+
 volumes:
-  postgres-data:
\ No newline at end of file
+  postgres-data:
+  redis-data:
+  redis-conf:
```

Comment:
no newline at the end of file
![image](https://user-images.githubusercontent.com/4482210/133456234-5c9222b4-ba80-489d-a1e6-a657929a4be0.png)